libs/vertexai/tests/integration_tests/test_embeddings.py

```diff
@@ -68,3 +68,9 @@ def test_warning(caplog: pytest.LogCaptureFixture) -> None:
         "Feb-01-2024. Currently the default is set to textembedding-gecko@001"
     )
     assert record.message == expected_message
+
+
+def test_langchain_google_vertexai_image_embeddings(tmp_image) -> None:
+    model = VertexAIEmbeddings(model_name="multimodalembedding")
```

Comment:
nits: should we add "multimodalembedding" to other tests maybe? to check how it works with text?

Reply:
@lkuligin I have some checks on the unittests section. In this multimodalembedding model can only be used for image embeddings, so I thought this integration tests would be enough for now.
In order to support multimodal for query and documents embeddings, we need some changes, as it only accepts a single str as input, instead of List[str]